FirstHW_JS/Task-1.js

```diff
@@ -0,0 +1,11 @@
+//Задача 1
+
+ const userNames = ["Петрик Ольга Іванівна", "Гнатюк Петро Антонович", "Рудко Андрій Опанасович"];
+    
+let initials = userNames.map(name => {
+    let splitedNames = name.split (" "); // [ [Петрик] [Ольга] [Іванівна], [Гнатюк] [Петро] [Антонович], [Рудко] [Андрій] [Опанасович] ]
+    let FIO = splitedNames.map(item => item[0].toUpperCase()); // [ [ 'П', 'О', 'І' ], [ 'Г', 'П', 'А' ], [ 'Р', 'А', 'О' ] ]
+    return FIO.join('.'); // [ 'П.О.І', 'Г.П.А', 'Р.А.О' ]
```

Comment:
```suggestion
    return name.split (" ").map(item => item[0].toUpperCase()).join('.');
```

Не треба створювати лишні змінні, цей код можна спростити